<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeaderEdit</class>
 <widget class="QDialog" name="HeaderEdit">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Header</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QSlider" name="slValue">
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSpinBox" name="sbValue">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HeaderEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HeaderEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
